config/routes.rb

```diff
@@ -37,6 +37,7 @@
   resources :patients, only: [:index, :new, :create, :show, :edit, :update, :new_group_member]
 
   resources :admin, only: [:index, :create_user]
+  get 'admin/users', to: 'admin#users'
```

Comment:
Did not test, but I think you can cover this in the above line with an additional `:users` in the resources array.

Reply:
This actually doesn't work. Made me read up more on Rails routing though. The reason it doesn't work is because it seems that only the default actions (:create, :edit, :update, etc.) can be put in `only:` which is really meant to be used for shallow-nesting of routes. 

I also found that in the line above this one, the `:new_group_member` also wasn't actually generating a route and we already had a route to that controller action below, so I removed it.

Reply:
Responding here for tracking (Adam replied in email):
Yeah the `:create_user` above I will be removing too. As far as a user_controller, we could break out the creating and editing functionality of users into a user_controller instead. Thoughts?

Reply:
Let's not break it out in this PR, but we should create a ticket to revist in the future.

Reply:
Sounds good. Done and also created new ticket: [SARAALERT-653](https://tracker.codev.mitre.org/browse/SARAALERT-653)